ckanext/canada/templates/internal/recombinant/create_pd_record.html

```diff
@@ -15,19 +15,24 @@
   <form method="POST" id="create_pd_record" novalidate>
 
     {% block errors %}
-      {%- if errors -%}
-        {%- set chromo = h.recombinant_get_chromo(resource_name) -%}
+      {%- if errors or error_summary -%}
         <div class="error-explanation alert alert-danger">
-          <p>{{ _('The form contains invalid entries:') }}</p>
-          <ul>
-            {% for field in chromo.fields -%}
-              {%- if field.datastore_id in errors %}
-                <li data-field-label="{{ key }}">{{
-                  h.recombinant_language_text(field.label) }}:
-                  {{ errors[field.datastore_id][0] }}</li>
-              {%- endif -%}
-            {%- endfor %}
-          </ul>
+          {% if error_summary %}
+            <p>{{ error_summary }}</p>
```

Comment:
this might result in double error messages if both error_summary and errors are present?

Reply:
In our logic of the view methods for update and create PD record, only either the `error` or `error_summary` will be set. So no issue here (for now at least)